index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- Navbar section -->
+    <nav id="navbar" class="nav">
+      <ul class="nav-list">
+        <li><a href="#welcome-section">About</a></li>
+        <li><a href="#projects">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+
+    <!-- Welcome section  -->
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+    <!-- Project Section -->
+    <section id="projects" class="projects-section">
+      <h2 class="projects-section-header">These are some of my projects</h2>
+      <div class="projects-grid">
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+          <a href="" class="project project-title">
+              <p class="project-title">"Tribute Page"</p>
+          </a>
+      </div>
+      <a href="#" class="btn btn-show-all">Show all</a>
+  </section>
```

Comment:
Good use of section and class names, keep it up

Reply:
Good work done